priv/repo/migrations/20170604172242_create_account_user.exs

```diff
@@ -2,14 +2,17 @@ defmodule Agoneum.Repo.Migrations.CreateAgoneum.Account.User do
   use Ecto.Migration
 
   def change do
-    create table(:account_users) do
+    create table(:users) do
```

Comment:
So account_users is in fact what phx.gen.schema gives you. Definitely not set in stone, just saying.